src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -1109,7 +1109,8 @@ static int vmr_info_query_op(struct platform_device *pdev,
 				goto done;
 			}
 			memcpy_from_device(xgq, address, info_data, info_size);
-			*cnt += sprintf(buf, "%s", info_data);
+			sprintf(buf, "%s", info_data);
+			*cnt = info_size;
```

Comment:
the original code is doing += and you change to =. why do you change the logic here? would it still work?